src/Renderer/LayeredMaterial.js

```diff
@@ -154,6 +154,12 @@ const LayeredMaterial = function LayeredMaterial(id) {
     // this.wireframe = true;
 
     this.colorLayersId = [];
+
+    if (__DEBUG__) {
+        this.uniforms.showOutline = { value: LayeredMaterial.prototype._showOutline };
```

Comment:
Yes. Note that currently it would be a configuration option of the **geometry** layer - because that's the way the shader work currently.
It would be nice to display border (with different color) per imagery layer, but again that's a different PR.